src/partials/header.html

```diff
@@ -0,0 +1,39 @@
+<div class="header">
+    <div class="container">
+        <img class="header__logo"
+            src="../images/logo 1.jpg"
+            alt="Logo IceCream"
+            width="199"
+            height="50"/>
+        <div class="header__hamburger">
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
+            <svg class="header__hamburger--icon">
+                <use href="../images/icons.svg$icon-hamburger"></use>
+            </svg>
+        </div>
+        <div class="header__title">
+            ice cream<br>made with<br><span class="header__title--passion">passion</span>
+        </div>
+        <button type="button">products</button>
+        <button type="button">how it's made?</button>
+        <img class="header__photo--icecream"
+            src="../images/header-icecream.jpg"
+            alt="icecream"
+            width="171"
+            height="410"/>
+    </div>
+</div>
```

Comment:
Brakuje mi ogólnie tych przycisków na górze - od "Home" do "Contacts" z wersji desktop. Nie wiem czy to najsensowniejsze podejście, ale w pracy domowej zrobiłem tak, że dopóki nie był to rozmiar desktop to były takie elementy display: none;
Chyba, że planujesz potem wziąć je z mobilnego menu, to wtedy w porządku. Nie wiem tylko czy to będzie proste, bo nie znajdują się wtedy te elementy w div z klasa header.